java/src/main/java/ai/rapids/cudf/CudaComputeMode.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright (c) 2019, NVIDIA CORPORATION.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package ai.rapids.cudf;
+
+/**
+ * This is the Java mapping of CUDA device compute modes.
+ */
+public enum CudaComputeMode {
+  cudaComputeModeDefault(0),
+  cudaComputeModeExclusive(1),
+  cudaComputeModeProhibited(2),
```

Comment:
Thee enums do not match the java naming convention, which is UPPER_CASE_WITH_UNDERSCORES. Also because all of them are under the `CudaComputeMode` class we don't need to prefix them all with `cudaComputeMode`

Could you please change them to

```
public enum CudaComputeMode {
    DEFAULT(0),
    EXCLUSIVE(1),
    PROHIBITED(2),
    EXCLUSIVE_PROCESS(3);
...
```

Reply:
I applied the UPPER_CASE_WITH_UNDERSCORES style.